com.zoho.Notebook.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: com.zoho.Notebook
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: notebook
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=wayland
```

Comment:
does it support wayland? if yes then it should be `fallback-x11` on the next line. Otherwise this line should be removed.

Reply:
It support. I will change.

Reply:
Ok, so before was working on Wayland, but after change to this way it stopped to work, so I guess it not works on Wayland. I changed to only X11.